example/lib/pages/custom_controller_page.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:better_player/better_player.dart';
+import 'package:better_player_example/pages/custom_controls/CustomControlsWidget.dart';
+import 'package:flutter/material.dart';
+
+class CustomControllerPage extends StatefulWidget {
+  @override
+  _CustomControllerPageState createState() => _CustomControllerPageState();
+}
+
+class _CustomControllerPageState extends State<CustomControllerPage> {
+  BetterPlayerController _betterPlayerControllerMaterial;
```

Comment:
Can you try do it with only one controller? User can change player style with 3 buttons.